pkgs/tools/admin/lego/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, fetchgit, buildGoPackage }:
+
+buildGoPackage rec {
+  name = "lego-${version}";
+  version = "0.4.2-pre-${lib.strings.substring 0 7 rev}";
```

Comment:
Since this revision is not released, can you please use for the version, the format `unstable-YYYY-MM-DD` as specified in https://nixos.org/nixpkgs/manual/#sec-package-naming